components/channel_members_dropdown/channel_members_dropdown.jsx

```diff
@@ -35,19 +35,29 @@ export default class ChannelMembersDropdown extends React.Component {
         this.state = {
             serverError: null,
             user: null,
-            role: null
+            role: null,
+            removing: false
         };
     }
 
     handleRemoveFromChannel() {
+        if (this.state.removing) {
+            return;
+        }
+
+        this.setState({removing: true});
+
         removeUserFromChannel(
             this.props.channel.id,
             this.props.user.id,
             () => {
                 this.props.actions.getChannelStats(this.props.channel.id);
```

Comment:
Looks like this doesn't reset `state.removing` to `false` on success. Is that a problem? Will it prevent me from removing multiple users?

Reply:
It won't affect multiple users, as each form is an individual component with internal state. 

I assume, once a user is removed the form is also removed, so the state doesn't need to be reset on success. But, to confirm, we should wait on @saturninoabril to confirm. :)